src/microbe_stage/editor/PatchMapDrawer.cs

```diff
@@ -33,7 +36,7 @@ public partial class PatchMapDrawer : Control
     /// <summary>
     ///   The representation of connections between regions, so we won't draw the same connection multiple times
     /// </summary>
-    private readonly Dictionary<Vector2I, Vector2[]> connections = new();
+    private readonly Dictionary<Vector2I, (Vector2[], bool, bool)> connections = new();
```

Comment:
I think this should use a named tuple here as otherwise it is very unclear what the two added booleans mean.